base/src/model.rs

```diff
@@ -1645,6 +1645,8 @@ impl Model {
                 column: cell.column,
             });
         }
+
+        self.garbage_collector().unwrap();
```

Comment:
We shouldn't call the gc on every evaluate call, this will make things extremely slow.
Also we shouldn't use `unwrap` in non test code as mentioned earlier.

You can use the gc when saving to disk or right before serializing to json.